packages/twenty-front/src/modules/ui/input/components/ImageInput.tsx

```diff
@@ -69,7 +69,7 @@ const StyledText = styled.span`
 `;
 
 const StyledErrorText = styled.span`
-  color: ${({ theme }) => theme.font.color.danger};
+  color: ${({ theme }) => theme.color.red};
```

Comment:
this should not be necessary if we keep font.color.danger = color.red, see comment below